components/Ink/src/MDCInkView.m

```diff
@@ -77,7 +77,8 @@ - (void)layoutSubviews {
   CGRect inkBounds = CGRectMake(0, 0, CGRectGetWidth(self.bounds), CGRectGetHeight(self.bounds));
 
   // When bounds change ensure all ink layer bounds are changed too.
-  for (CALayer *layer in self.layer.sublayers) {
```

Comment:
Will this be a problem? It doesn't seem like changing the bounds will modify the `sublayers` array.

Reply:
Done.